dev/devicelab/lib/framework/ios.dart

```diff
@@ -135,3 +140,60 @@ Future<void> removeIOSimulator(String deviceId) async {
     );
   }
 }
+
+Future<bool> runXcodeTests(String projectDirectory, String deviceId, String testName) async {
```

Comment:
This logic was moved from `native_ui_tests_ios32` so it can be used by multiple tests.